### DISCUSSION TITLE
Option to select a default directory on first launch

### DISCUSSION THREAD

**Discussion by tuszmak** [ppy/osu#21749] [Ideas]
Currently osu!lazer puts its files into the C: drive, and I have to manually change it to switch it to another drive (which is larger and I can store it without issue)

**Comment by integerrr** [comment]
But to launch the game for the first time requires the game files to be initialised and stored *somewhere*? Unless you're suggesting a dialog to choose where the game is install before the game launches for the first time?

**Reply by tuszmak** [reply]
Oh, yes. I meant a dialog before initialization

**Comment by cheeezie** [comment]
I'd agree with that, the stable version has such option but lazer still doesn't.
